docs/resources/app.md

```diff
@@ -0,0 +1,114 @@
+---
+subcategory: "Apps"
+---
+# databricks_app Resource
+
+-> This feature is in [Public Preview](https://docs.databricks.com/release-notes/release-types.html).
+
+[Databricks Apps](https://docs.databricks.com/en/dev-tools/databricks-apps/index.html) run directly on a customer’s Databricks instance, integrate with their data, use and extend Databricks services, and enable users to interact through single sign-on. This resource creates the application but does not handle app deployment, which should be handled separately as part of your CI/CD pipeline.
+
+## Example Usage
+
+```hcl
+resource "databricks_app" "this" {
+  name             = "my-custom-app"
+  description      = "My app"
+  resources = [{
```

Comment:
Having resources as array isn't really nice IMHO - individual `resource` blocks are easier to use